src/scraper/scraper.h

```diff
@@ -70,7 +70,7 @@ int64_t SCRAPER_CMANIFEST_RETENTION_TIME = 48 * 3600;
 bool SCRAPER_CMANIFEST_INCLUDE_NONCURRENT_PROJ_FILES = false;
 double MAG_ROUND = 0.01;
 double NEURALNETWORKMULTIPLIER = 115000;
-double CPID_MAG_LIMIT = 32767;
+double CPID_MAG_LIMIT = NN::Magnitude::MAX;
```

Comment:
Yes I agree with this. The default value (absent a protocol override) should be this.

Reply:
We should probably put a clamp in ScraperApplyAppCacheEntries() to constrain a mag limit value specified in the protocol to NN:Magnitude::MAX, to ensure we don't break things if someone happens to put a CPID_MAG_LIMIT above the hard coded constant. Thoughts?

Reply:
I think that's a good idea. If we do, we may want to add reasonable boundaries to some of the other protocol configuration items as well, though.